docs/reference/config.md

```diff
@@ -316,6 +316,8 @@ Each mapping in the `plugins` collection has the following keys:
 
 - `name`:
   - The name of this plugin. Required
+- `env`
+  - A list of environment variables to pass to the plugin. By default, no environment variables are passed.
```

Comment:
There is the `SQLC_VERSION` var which is added (not from the actual shell environment though). That should be documented somewhere, but probably not here though.